instrument/D2B_Parameters_2018-03-01.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<parameter-file instrument="D2B" valid-from="2018-03-01 23:59:59">
+	<component-link name="D2B">
+
+		<!-- The detector name for the height axis used in SumOverlappingTubes -->
+		<parameter name="detector_for_height_axis" type="string">
+			<value val="tube_1" />
+		</parameter>
+
+		<!-- The flag for mirroring the angles in SumOverlappingTubes -->
+		<parameter name="mirror_scattering_angles" type="bool">
+			<value val="True" />
+	  </parameter>
+
+		<!-- Number of pixels to trim from the top and the bottom of the tubes when
+		calculating the chiˆ2 in the detector efficiencies in case of autoiterations -->
+		<parameter name="pixels_to_trim" type="int">
+			<value val="28" />
+	  </parameter>
+
+		<!-- The chiˆ2/NdoF threshold for termination of autoiterations
+		in detector efficiency calculation -->
+		<parameter name="chi2_ndof" type="float">
+			<value val="0.01" />
+	  </parameter>
+
+		<!-- The central pixel indices of the tubes-->
+		<parameter name="tube_centers" type="string">
```

Comment:
Is there a reason for setting these here, rather than just having an IDF with these values in?